tests/data_import/test_upload.py

```diff
@@ -27,7 +27,11 @@ def test_missing_fields_raises_exception(standardized_data_upload_blob,
 
     mock_file = standardized_data_upload_blob['standardized_file']
 
-    rv = admin_client.post(reverse('upload-standardized-file'),
+    # rv = admin_client.post(reverse('upload-standardized-file'),
+    #                        data=standardized_data_upload_blob,
+    #                        files={'standardized_file': mock_file})
+
+    rv = admin_client.post('/admin/',
```

Comment:
This change is on the right track, but you'll need to POST to a more specific URL. I would recommend opening [the network panel](https://developers.google.com/web/tools/chrome-devtools/network) in your browser console, then using the admin interface to upload a test file. Use the network panel, specifically [the headers and post data from the resource details tab](https://developers.google.com/web/tools/chrome-devtools/network#details), to figure out what URL data is posted to. Make sense?